packages/block-editor/src/hooks/layout-child.js

```diff
@@ -30,6 +43,26 @@ function useBlockPropsChildLayoutStyles( { style } ) {
 			css = `${ selector } {
 				flex-grow: 1;
 			}`;
+		} else if ( columnSpan ) {
+			css = `${ selector } {
+				grid-column: span ${ columnSpan };
+			}`;
+		}
+		/**
+		 * If parentColumnWidth is set, the grid is responsive
+		 * so a container query is needed for the span to resize.
+		 */
+		if ( columnSpan && parentColumnWidth ) {
```

Comment:
Should we also check that `containerQueryValue` and `parentColumnUnit` were successfully converted?